prisma/schema.prisma

```diff
@@ -175,4 +176,20 @@ model OrderReturn {
   user            User     @relation(fields: [userId], references: [id])
   product         Product  @relation(fields: [productId], references: [id])
   order           Order    @relation(fields: [orderId], references: [id])
+}
```

Comment:
@kencas Changing this model resulted in a series of error in the project. Please kindly review this once more. 
	

- **Error validating: This line is not an enum value definition."
- This line is not an enum value definition
- Error validating: An enum must have at least one value.",**